Shop.DAL/Interfaces/IRepositoryManager.cs

```diff
@@ -0,0 +1,11 @@
+namespace Shop.DAL.Interfaces;
+
+public interface IRepositoryManager
```

Comment:
it is better to call it IUnitOfWork

Reply:
Yes, it can be called IUnitOfWork since it is name pattern that used here, but RepositoryManager name was chosen as it more meaningful and descriptive. I'll change it, but is it really better?